core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7535,6 +7558,8 @@ public static final class Name {
     public static final String MASTER_METADATA_SYNC_UFS_PREFETCH_TIMEOUT =
         "alluxio.master.metadata.sync.ufs.prefetch.timeout";
     public static final String MASTER_METASTORE = "alluxio.master.metastore";
+    public static final String MASTER_INODE_METASTORE = "alluxio.master.metastore.inode";
+    public static final String MASTER_BLOCK_METASTORE = "alluxio.master.metastore.block";
```

Comment:
pls keep the property names aligned
```suggestion
    public static final String MASTER_METASTORE_INODE = "alluxio.master.metastore.inode";
    public static final String MASTER_METASTORE_BLOCK = "alluxio.master.metastore.block";
```

Reply:
+1